Name change from omSupply to Open mSupply

fergie-nz opened the pull request:
Fixes #2471 

# 👩🏻‍💻 What does this PR do? 
Name change from omSupply to Open mSupply

# 🧪 How has/should this change been tested? 
I think this will only change superficial text to open mSupply.
I would also love to test, but I don't know exactly how! 
If anyone is willing to show me how this works, I'd love to hear from you :)

mark-prins reviewed (changes requested):
The desktop shortcut name is still omSupply:

![desktop-shortcut](https://github.com/msupply-foundation/open-msupply/assets/9192912/e47d0848-98d9-4e0d-8210-2ea2a8af9864)

(that's in omsupply_desktop.suf::3125)

The app bar for server, desktop and standalone have omsupply:

![desktop-upgrader](https://github.com/msupply-foundation/open-msupply/assets/9192912/5b88f88b-d3f5-49e9-996a-49b2d750789c)

![server-install](https://github.com/msupply-foundation/open-msupply/assets/9192912/21bbf758-fbaf-4676-a938-996e0b7ef901)

![standalone-install](https://github.com/msupply-foundation/open-msupply/assets/9192912/a954fbc9-f4a5-419e-b5be-66e3a1fbe478)

The server database settings are squashed up:

![server](https://github.com/msupply-foundation/open-msupply/assets/9192912/fe9e1052-df49-4493-84cd-b0dfd80f32ce)


I've made the changes in [this branch](https://github.com/msupply-foundation/open-msupply/compare/2471-rename-to-Open-mSupply...2471-rename-to-Open-mSupply(after-testing)) - can you have a look through?

I've renamed the window titles (3 needed updating, 1 was fine) and checked the install and upgrade versions (they're both different 😜) renamed the shortcut and adjusted the layout on the database selection screen.

fergie-nz commented:
> The desktop shortcut name is still omSupply:
> 
> ![desktop-shortcut](https://private-user-images.githubusercontent.com/9192912/288578052-e47d0848-98d9-4e0d-8210-2ea2a8af9864.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5MTQzNDEsIm5iZiI6MTcwMTkxNDA0MSwicGF0aCI6Ii85MTkyOTEyLzI4ODU3ODA1Mi1lNDdkMDg0OC05OGQ5LTRlMGQtODIxMC0yZWEyYThhZjk4NjQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMDdUMDE1NDAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWM5MWZlZDYyODczYTljZWNmZmRhNTM0Y2IxZmMyZGJhOTM2NjM1NTE1MjA0NjY4N2VhNjM1OWVjM2RlNDFjYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.4hMJFPVovsa7GvbopHltLb3GtSC6JBlibe9_Q-90VVw)
> 
> (that's in omsupply_desktop.suf::3125)
> 
> The app bar for server, desktop and standalone have omsupply:
> 
> ![desktop-upgrader](https://private-user-images.githubusercontent.com/9192912/288578058-5b88f88b-d3f5-49e9-996a-49b2d750789c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5MTQzNDEsIm5iZiI6MTcwMTkxNDA0MSwicGF0aCI6Ii85MTkyOTEyLzI4ODU3ODA1OC01Yjg4Zjg4Yi1kM2Y1LTQ5ZTktOTk2YS00OWIyZDc1MDc4OWMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMDdUMDE1NDAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDY1YWUyOTc0NWY0ZTlmZDM2MDRkMzRhNDBkNGZjYjQ3MWE5ZDc1MjkxZWM5MmRmZDZmZGU5NTdjYjRiOTRhNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.JByBGLLh3OvWXzxqXLJeSVS4SpCoBGzdu6CdlVFYPfU)
> 
> ![server-install](https://private-user-images.githubusercontent.com/9192912/288578061-21bbf758-fbaf-4676-a938-996e0b7ef901.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5MTQzNDEsIm5iZiI6MTcwMTkxNDA0MSwicGF0aCI6Ii85MTkyOTEyLzI4ODU3ODA2MS0yMWJiZjc1OC1mYmFmLTQ2NzYtYTkzOC05OTZlMGI3ZWY5MDEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMDdUMDE1NDAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDE4ZmU2OWNlOGUwN2E1MzRhYzAyMmQ5ZjUwZGM0OWI2ZTE1ZGY1ZTM2NjdiNWFlMzE4NGRmM2Y2ZThlMDNjNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.iBuN3CMtCNSgZvCuI-K5Gz6cqVxutrU4kPWaNElHhzE)
> 
> ![standalone-install](https://private-user-images.githubusercontent.com/9192912/288578066-a954fbc9-f4a5-419e-b5be-66e3a1fbe478.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5MTQzNDEsIm5iZiI6MTcwMTkxNDA0MSwicGF0aCI6Ii85MTkyOTEyLzI4ODU3ODA2Ni1hOTU0ZmJjOS1mNGE1LTQxOWUtYjViZS02NmUzYTFmYmU0NzgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMDdUMDE1NDAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzdiZmM0MDk2NzQ1ZTdjMTVmYTc2ODcwYTE3MDYwYjdjMTViZjRiMDUxOGVkZDcxMmU4ZmJkYzU0YTFhZWFiYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.A2MbjB2OStF-DhPXy3nwqNhgivLJofK5TAGXOhmLvR4)
> 
> The server database settings are squashed up:
> 
> ![server](https://private-user-images.githubusercontent.com/9192912/288578063-fe9e1052-df49-4493-84cd-b0dfd80f32ce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5MTQzNDEsIm5iZiI6MTcwMTkxNDA0MSwicGF0aCI6Ii85MTkyOTEyLzI4ODU3ODA2My1mZTllMTA1Mi1kZjQ5LTQ0OTMtODRjZC1iMGRmZDgwZjMyY2UucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMDdUMDE1NDAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWNkMzYzMDk0MzcxYzVjOTQzOTcyZjQ5MmM1NGFmY2NhZmUzMDgzNTkwMjA0ODRhYmRjYmZiN2Y2YjYzYmY1NCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.0F7wecqO0LOeLLZAO6HcdQ1tanJpQe67Np8_ekkPKT4)
> 
> I've made the changes in [this branch](https://github.com/msupply-foundation/open-msupply/compare/2471-rename-to-Open-mSupply...2471-rename-to-Open-mSupply(after-testing)) - can you have a look through?
> 
> I've renamed the window titles (3 needed updating, 1 was fine) and checked the install and upgrade versions (they're both different 😜) renamed the shortcut and adjusted the layout on the database selection screen.

That is amazing! Thanks Mark,

I haven't been able to open these, however, so I can't test it.
